src/Our.Umbraco.Ditto/ComponentModel/Attributes/DittoLazyAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Our.Umbraco.Ditto.ComponentModel.Attributes
+{
+    /// <summary>
+    /// The Ditto lazy property attribute.
+    /// Used for specifying that Ditto should lazy load this property.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Class, AllowMultiple = false)]
```

Comment:
I'd expect it to work in the same way as other attributes. I.E if I mark an enum with the `EnumAttribute`  then every time I declare that enum type as a property I know the processor will kick in without me having to declare the attribute again. Same as with the `UmbracoPickerAttribute`. I declare any base classes with that attribute so I can use them as properties at any time and always have a working picker.